js/templates/testProfile.html

```diff
@@ -23,33 +23,33 @@
   <label>
     Social Accounts:
   </label>
+  <div>
```

Comment:
These are mostly white-space changes. I did change this quite a bit, but ended up reverting it via copy and paste.